src/util/time.h

```diff
@@ -40,9 +40,16 @@ int64_t GetTimeMicros();
 /** Returns the system time (not mockable) */
 int64_t GetSystemTimeInSeconds(); // Like GetTime(), but not mockable
 
-/** For testing. Set e.g. with the setmocktime rpc, or -mocktime argument */
+/**
+ * For testing. Set e.g. with the setmocktime rpc, or -mocktime argument
+ *
+ * @param[in] nMockTimeIn Time in seconds.
```

Comment:
Probably more of a hassle, but another way to document would be to use `std::chrono` types with explicit units.

Reply:
yeah, I took a quick look at changing the signature to a chrono type, but ended up taking the efficient/lazy way for now. there are a couple tweaks that would need to be made to switch it over, nothing difficult but not currently at the top of my priority list

so I thought leaving a comment was the smallest way to help :)

Reply:
in the latest push, I updated the function signature of `GetMockTime`, with updated call sites. I also overloaded `SetMockTime` to take in chronos. This does mean I introduced another function that doesn't get used until #21061, so I can remove if you'd prefer.